client/src/components/Loader/style.scss

```diff
@@ -0,0 +1,25 @@
+.lds-dual-ring {
+  display: inline-block;
+  width: 128px;
+  height: 128px;
+  margin-top: 12rem;
```

Comment:
Choose only one CSS unit and use through the project. Relative units are always better than pixels, so have in mind to use one of them, never both.